thirdparty/include/darts.h

```diff
@@ -283,6 +283,12 @@ class DoubleArrayImpl {
       std::size_t max_num_results, std::size_t length = 0,
       std::size_t node_pos = 0) const;
 
+  // Performs commonPrefixSearch() for every possible one-element-deleted `key`.
```

Comment:
Do you mean only one element (character) can be deleted from the original spelling?
How does `correction_level` work?

If not limited to one element, call it "`key` with deleted elements",

...and possibly name the function `commonPrefixSearchWithDeletes`. Can be confused because it isn't the "common prefix" that gets deleted.

Reply:
This is now deprecated.

Reply:
I found it's difficult to perform multiple deleted symmetric search other than 1 to fit the trie query.
Maybe we can only do correction in 2 edit distance level.

Reply:
It's terrible to make 2 errors in a pinyin syllable. We can now focus on making 1 edit.